src/game_api/level_api.hpp

```diff
@@ -498,8 +498,8 @@ struct LevelGenSystem
         };
     };
     uint8_t flags;
-    uint8_t unknown39; // also flags?
-    uint8_t unknown40; // also flags?
+    uint8_t flags2;
+    uint8_t flags3;
```

Comment:
Is it just one 32bit flags? Or maybe we have just combined multiple 8bit flags to 32bit flags before, but it doesn't really matter...

Reply:
Oh I assumed there was just automatic padding added by aligning the next thing and there's actually 32 bits there.

Reply:
I would still use 32bit, it doesn't do any hard really, unless you know the specifics of each flags variable

Reply:
I don't understand how it could be 32 bit when there are only 24 bits here.

I do know the specifics of each flag variable, I added the details to the plugin, but not sure how to name those for OL.